### PR TITLE
Updated bootstrap-notify to v3.1.3

### PR DESCRIPTION
**Pull request by robert-voica**

### CODE REVIEW

**Comment by vvakame** [comment]
*bootstrap-notify/bootstrap-notify.d.ts*

to author(@niemyjski). could you review this PR?
:+1: or :-1:?

check list

* [ ] pass the Travic-CI test?

File: bootstrap-notify/bootstrap-notify.d.ts [hunk 1]
```diff
@@ -1,68 +1,62 @@
-// Type definitions for bootstrap-notify
-// Project: https://github.com/Nijikokun/bootstrap-notify
-// Definitions by: Blake Niemyjski <https://github.com/niemyjski/>
```
Comment (vvakame) [inline bootstrap-notify/bootstrap-notify.d.ts]: please keep original author name

File: bootstrap-notify/bootstrap-notify.d.ts [hunk 2]
```diff
@@ -1,68 +1,62 @@
...
+	notify(opts: INotifyOptions, settings?: INotifySettings): INotifyReturn;
+	notifyDefaults(settings: INotifySettings): void;
+	notifyClose(): void;
+	notifyClose(command: string): void;
 }
 
-interface MessageOptions {
-    html?: string;
-    text?: string;
+interface INotifyOptions {
```
Comment (vvakame) [inline bootstrap-notify/bootstrap-notify.d.ts]: prefix `I` is bad pattern by Microsoft

**Comment by robert-voica** [comment]
@vvakame, I have done the requested changes

**Comment by vvakame** [comment]
@robert-voica thanks mate!

**Comment by KarolBuchta** [comment]
@robert-voica Hi, i am not sure, if i simply create an issue here, if the message reaches you. But the boostrap-notify INotificationSettings is missing something, that can be used in js.

offset in the settings can also be an object:

offset?:{
  x? :number,
  y? :number
}

See: https://github.com/mouse0270/bootstrap-notify/blob/master/bootstrap-notify.js#L112

I'll post on the original repo too.

**Pull request merged**